docs/users-guide/04-asking-questions.md

```diff
@@ -107,30 +143,68 @@ Some grouping columns will give you the option of choosing how big or small to m
 
 ![Multiple groupings](./images/notebook/histogram-bins.png)
 
-Once you're done setting your metrics and groupings, click Done to close the Summarize sidebar and see your results in all their glory.
+Once you're done setting your metrics and groupings, click **Done** to close the Summarize sidebar and see your results in all their glory.
```

Comment:
This doc feels like it's combining the "simple mode" and notebook experiences now in a couple places in a potentially confusing way. E.g., if you've been following along and are trying to summarize in the notebook, there is no "Summarize sidebar."